modules/seqexec/engine/src/main/scala/seqexec/engine/Event.scala

```diff
@@ -18,20 +19,16 @@ final case class EventSystem(se: SystemEvent) extends Event[Nothing]
 
 object Event {
 
-  def start[D<:Engine.Types](id: Observation.Id, user: UserDetails, clientId: ClientID): Event[D] = EventUser[D](Start(id, user.some, clientId))
+  def start[D<:Engine.Types](id: Observation.Id, user: UserDetails, clientId: ClientID, f: D#StateType => Boolean): Event[D] = EventUser[D](Start[D](id, user.some, clientId, f))
   def pause[D<:Engine.Types](id: Observation.Id, user: UserDetails): Event[D] = EventUser[D](Pause(id, user.some))
   def cancelPause[D<:Engine.Types](id: Observation.Id, user: UserDetails): Event[D] = EventUser[D](CancelPause(id, user.some))
-  def load[D<:Engine.Types](id: Observation.Id, sequence: Sequence): Event[D] = EventUser[D](Load(id, sequence))
```

Comment:
Why are these gone? `load`, `unload`, `setObserver`

Reply:
They are implemented differently, and don't require an event anymore.